guides/v2.3/mrg/ce/Version.md

```diff
@@ -2,6 +2,33 @@
 title: Magento_Version module
 ---
 
-{% include mrg/note.md %}
+`Magento\Version` allows to get Magento version and edition by HTTP GET request.
 
-Magento\Version Allows to get Magento version and edition by HTTP GET request
+**Request**
+```text
+http://<magento2-store>/magento_version
+```
+
+**Response**
+```text
+Magento/2.3 (Community)
+```
+
+Run the command below to get a full Magento version.
+
+**Command**
+```text  
+php bin/magento --version
+```
+
+**Response**
+```text
+Magento CLI version 2.2.5
```

Comment:
```suggestion
Magento CLI version 2.3.0
```

Reply:
> Suggested change
> Magento CLI version 2.2.5
> Magento CLI version 2.3.0

Good point, @osrecio  👍 

**fixed**

Thank you!